### ISSUE TITLE
Enable collecting metrics with a config option

### ISSUE THREAD

**Issue by adambabik** [status-im/status-go#1224]
# Problem

Currently, status-go can collect very useful metrics when run with `-metrics` flag. However, after we introduced JSON config files, it should be possible to enable metrics with an entry in the config file.

# Acceptance Criteria

It's possible to collect metrics using a config option in a JSON file and retrieve them with `debug_metrics`.

**Comment by divan** [comment]
@adambabik can you also write what exactly the use cases of using status-go as a library (and why they'd need metrics exposed). I assume it's Status Desktop mostly, right?

**Comment by adambabik** [comment]
@divan sorry, I got it super confused. I fixed the description. What I meant is to collect metrics with a config option. Currently, it's possible only with a command line flag `-metrics`.

However, this will cover a case of collecting metrics when status-go is run as a library (like in status-react or Status Desktop case). I believe it's gonna be very useful to debug what's going on with a embedded node (does it receive any traffic from disc/whisper/eth protocols, size of the traffic etc).
